test/ext/ws.js

```diff
@@ -749,6 +749,24 @@ describe("web-sockets extension", function () {
                 window.document.removeEventListener("htmx:wsOpen", handler)
             }
         })
+        it('sends message if no sending element is provided', function (done) {
```

Comment:
@alexpetros This test is the one which would currently fail; no sending element would result in the message being gobbled up.